dependencies/dependencies.xml

```diff
@@ -98,11 +98,11 @@
       <url>https://github.com/Hopsan/hopsan_qt_mingw64/releases/download/v5.9.9/qt-5.9.9-x64-5.4.0-release-posix-seh-rt_v5-rev0-webkit.7z</url>
     </releasefile>
   </dependency>
-  
+
   <dependency name="fmi4c">
-	<releasefile sha1="fd1bce2369c2a3833de644e08feef2886f013b6c">
-		<url>https://github.com/robbr48/fmi4c/archive/refs/tags/v1.0-alpha.2.zip</url>
-	</releasefile>
+    <releasefile sha256="518e7249549e56630b42d5e4440c05644e3580ac7ac97c793c952c6489c30bd8">
```

Comment:
Is this supposed to be changed back when the fmi4c PR has been merged?

Reply:
I made a clone of this branch in my fork, because I had to rebase. I have additional commits there.
I have run into problems trying to use fmi4c as a dynamic library (on windows) so I think that we should build and use it as a static library. That is built into the defaultLibrary.

An alternative would be to include the FMIWrapper component with HopsanCore, like an internal component like "Subsystem" and "SystemPort". But I am not sure if that will work.

If you rebase this branch on master, I can reapply my additional commits, or I create a PR to this branch from my branch.
But it needs to be rebased. (I can do that as well if you want). But I did not want to do that as I did not know if you had work in progress.